BinaryEncoding.md

```diff
@@ -170,52 +223,79 @@ The function section _declares_ the signatures of all functions in the
 module (their definitions appear in the [code section](#code-section)).
 
 | Field | Type | Description |
-| ----- |  ----- | ----- |
+| ----- | ---- | ----------- |
 | count | `varuint32` | count of signature indices to follow |
 | types | `varuint32*` | sequence of indices into the type section |
 
 ### Table section
 
 ID: `table`
 
-The table section defines the module's
-[indirect function table](AstSemantics.md#calls).
+The encoding of a [Table section](Modules.md#table-section):
 
 | Field | Type | Description |
-| ----- |  ----- | ----- |
-| count | `varuint32` | count of entries to follow |
-| entries | `varuint32*` | repeated indexes into the function section |
+| ----- | ---- | ----------- |
+| element_type | `varuint7` | `0x40`, indicating [`anyfunc`](AstSemantics.md#table) |
+| | `resizable_definition` | see [above](#resizable_definition) |
 
 ### Memory section
 
 ID: `memory`
 
-The memory section declares the size and characteristics of the memory
-associated with the module.
+The encoding of a [Memory section](Modules.md#linear-memory-section) is simply
+a `resizable_definition`:
 
 | Field | Type | Description |
-| ----- |  ----- | ----- |
-| initial | `varuint32` | initial memory size in 64KiB pages |
-| maximum | `varuint32` | maximum memory size in 64KiB pages |
-| exported | `uint8` | `1` if the memory is visible outside the module |
+| ----- | ---- | ----------- |
+| | `resizable_definition` | see [above](#resizable_definition) |
+
+Note that the initial/maximum fields are specified in units of 
+[WebAssembly pages](AstSemantics.md#linear-memory).
+
+### Global section
```

Comment:
That's right; that's how it works for all the importable definitions (functions, memories, tables, globals) as well as arguments+locals.  This is documented in Modules.md in the "X index space" sections which I've tried to link to, where appropriate, from this doc.  But I could try to add a few more links.